src/components/Toaster/Toast/Toast.scss

```diff
@@ -192,9 +207,17 @@ $block: '.#{$ns}toast';
     }
 }
 
+// with extra space for icon placement
 @keyframes toast-set-indents {
     to {
         margin-bottom: 10px;
-        padding: 18px;
+        padding: 16px 16px 16px 48px;
```

Comment:
It's better to not animate "reflow" properties, instead it can be done with `transform`